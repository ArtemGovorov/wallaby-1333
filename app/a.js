import React from 'react';

export const Entry = () => {
  return (
    <div>
      Two
    </div>
  );
};
