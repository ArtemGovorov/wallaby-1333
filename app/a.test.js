import React from 'react';
import {mount, configure} from 'enzyme';
import toJson from 'enzyme-to-json';
import Adapter from 'enzyme-adapter-react-16';

configure({adapter: new Adapter()});


import {Entry} from './a';

it('handles Entry', () => {
  const wrapper = mount(<Entry/>);

  expect(toJson(wrapper)).toMatchSnapshot();
});
