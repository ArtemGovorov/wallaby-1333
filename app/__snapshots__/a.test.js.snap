// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`handles Entry 1`] = `
<Component>
  <div>
    Two
  </div>
</Component>
`;
